### PR TITLE
Italian translation fix

### PR DESCRIPTION
**Pull request by xfarrow**
### Feature Preview
* This pull request fixes an incorrect Italian translation

#### PR Checklist

- [X] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [X] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [X] All existing tests are passing.

### CODE REVIEW

**Review by MayurSMahajan — COMMENTED** [review]
Hi @xfarrow, thank you for contributing, can I request you to add some more missing italian translations in AppFlowy.
These are few I found to be missing:
 - Feedback (in the question bubble)
 - Theme Mode (in the appearance settings or aspetto settings)
 - Sign in with (in the User or Utente settings)
 - Shortcuts (in the settings, last tab)
 - Command, Key binding, and Reset to Keybindings inside the Shortcuts settings
 
 Thank you for helping out.

**Comment by xfarrow** [comment]
Hi @MayurSMahajan of course I can! The only thing is that the name of the shortcuts (e.g. "Move cursor down and select one line") seems to be outside of the `.json` file, hardcoded. Am I missing something?

**Comment by MayurSMahajan** [comment]
@xfarrow Thanks for adding new translations. 
Regarding the shortcuts:
I only want you to translate the Words "Shortcut", "Commands", "KeyBinding", and "Reset To Default",
The individual shortcuts like "Move cursor down .." are not supposed to be localized.

So right now our settings in Italian looks like this:
image

We only want to translate the labels which are underlined with red.

**Comment by xfarrow** [comment]
@MayurSMahajan Perfect then it's everything in the Pull Request already.

File: frontend/resources/translations/it-IT.json [hunk 1]
```diff
@@ -172,17 +173,17 @@
...
-      "open": "aprire le impostazioni",
+      "open": "Apri le impostazioni",
       "user": "Utente",
       "files": "File",
       "supabaseSetting": "Impostazione Supabase"
     },
     "appearance": {
       "themeMode": {
-        "label": "Theme Mode",
+        "label": "Tema",
```
Comment (MayurSMahajan) [inline frontend/resources/translations/it-IT.json:183]: we already use Tema for the word "Theme", is there something more appropriate for "Theme Mode"?

**Review by MayurSMahajan — CHANGES_REQUESTED** [review]
Amazing work!

File: frontend/resources/translations/it-IT.json (hunk 1 above)
Comment (Xazin) [reply frontend/resources/translations/it-IT.json:183]: `Luminosità del tema` (Theme brightness) or `Modalità tema` (Theme mode)

Maybe one of these? I'm no italian though, but google uses the second one, modalità tema in one of their support answers: 
https://support.google.com/accessibility/android/answer/6151800?hl=it

Comment (xfarrow) [reply frontend/resources/translations/it-IT.json]: Hi, as an Italian myself I felt "Tema" to be good, but "Modalità tema" is good as well, just updated the string to the latter.

**Review by MayurSMahajan — APPROVED** [review]

**Pull request merged**